ui-library-add-call-recording/api/listRecordings.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Contoso.Transfer;
+
+namespace Contoso
+{
+    public static class ListRecordings
+    {
+        [FunctionName("listRecordings")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            string serverCallId = req.Query["serverCallId"];
+            if (string.IsNullOrEmpty(serverCallId))
+            {
+                return new BadRequestObjectResult("`serverCallId` not set");
+            }
+
+            var storageClient = new BlobStorage(log, Settings.GetRecordingStoreConnectionString(), Settings.GetRecordingStoreContainerName());
+            var response = new ListRecordingsResponse
+            {
+                Blobs = storageClient.GetBlobs($"call_{serverCallId}")

```

Comment:
C# question, the docs say this is async:
![image](https://user-images.githubusercontent.com/2684369/152568660-52aa9208-e655-40c6-9ab6-fa2beb313cef.png)

But the function has a sister function `GetBlobsAsync` that says the same thing...

Is `GetBlobs` still doing async requests but it force waits here synchronously until the async requests are finished and collected?